net/freeradius/src/opnsense/service/templates/OPNsense/Freeradius/mods-enabled-ldap

```diff
@@ -70,15 +70,15 @@ ldap {
         user {
                 base_dn = "${..base_dn}"
 {%     if helpers.exists('OPNsense.freeradius.ldap.user_filter') and OPNsense.freeradius.ldap.user_filter != '' %}
-                filter = '{{ OPNsense.freeradius.ldap.user_filter }}'
+                filter = "{{ OPNsense.freeradius.ldap.user_filter }}"
```

Comment:
Since _OPNsense.freeradius.ldap.user_filter_ can contain a string like (uid=%{%{Stripped-User-Name}:-%{User-Name}}) and this string would then be inside single quotes, freeradius does not replace these variables and the ldap filter always fails